knpu/episode3/protected-visibility.md

```diff
@@ -34,7 +36,11 @@ of the class you're working in.
 Now, there is another designation between `private` and `public` which is called `protected`. 
 `Protected` works exactly like `private` except that subclasses can access it, so when we change
 it here the error goes away. Cool! Let's do a temporary fix for the error we're getting by making
-all of these things `protected`. Everything in our `RebelShip` file looks happy again so let's refresh.
```

Comment:
Maybe we should add a code block after line 26 above, after this sentence:

>
The answer has to do with this word `private` in front of `$name`.

We would show the private `$name` property in the ship class